### DISCUSSION TITLE
Not having permission to push branch.

### DISCUSSION THREAD

**Discussion by shoryakaushik** [vishal-codes/react-widgets#65] [General]
Shows "Permission to vishal-codes/react-widgets.git denied to shoryakaushik." when I push a branch to the repo. Please allow me. @vishal-codes

**Comment by vishal-codes** [comment]
> Shows "Permission to vishal-codes/react-widgets.git denied to shoryakaushik." when I push a branch to the repo. Please allow me. @vishal-codes

umm I guess I m late for this right?😅
